extra/setup.py

```diff
@@ -49,7 +49,7 @@ def create_version_file():
     'gluoncv>=0.9.1,<1.0',
     'scipy>=1.3.3,<1.5.0',
     'graphviz<0.9.0,>=0.8.1',
-    'bokeh',
+    'bokeh<=2.2.2',
```

Comment:
Why?

Reply:
I met a dependency conflict problem reported by CI so I added this.

Reply:
I turns out that this is not needed .